### ISSUE TITLE
`stencil push` -- Can't select which themes to delete with Git Bash

### ISSUE THREAD

**Issue by harnerdesigns** [bigcommerce/stencil-cli#395]
![image](https://user-images.githubusercontent.com/20668329/41258353-1271641a-6d8d-11e8-9a35-68b8a73068cf.png)

### Expected behavior
 1. `stencil push`
 2. "warning -- You have reached your upload limit.  In order to proceed, you'll need to delete at least one theme." 
 3. Select which theme to delete
 4. Command Uploads.

### Actual behavior
 1. `stencil push`
 2. "warning -- You have reached your upload limit.  In order to proceed, you'll need to delete at least one theme."
 2.  Slam up, down, left and right keys
 3. Hit Spacebar a bunch
 4. ...
 5. Ctrl + C to end process
 6. Come to github looking for other similar issues.
 7. Post new issue.

### Steps to reproduce behavior

I'm using Windows 10, git version 2.17.1.windows.2, Stencil 1.15.1 and the theme was based off of Cornerstone.  I'm able to `stencil bundle` and upload the theme via the Store Dashboard, but I can't seem to select which theme to delete and finish the upload. I assume it's an issue with Git Bash.

**Comment by harnerdesigns** [comment]
However, with Windows PowerShell the command works as expected and I'm able to select a theme or multiple themes to delete and it then uploads perfectly.

**Comment by steve-ross** [comment]
There is also some wonky-ness going on with theme uploads... seems they lowered the limit of the number of themes you can have to 5? Sucks when you can't delete 3 of them.

I wish the command for push wasn't using Async.waterfall so it would keep uploading when you delete one instead of failing...

**Comment by jeremysimmons** [comment]
@harnerdesigns I'm going to steal your actual behavior block for future issues

**Comment by harnerdesigns** [comment]
@steve-ross We've currently got 20 themes uploaded to our store so, at least for us, the limit is definitely higher than 5.

@jeremysimmons by all means go ahead!

**Comment by steve-ross** [comment]
@harnerdesigns yeah, we used to have 25 & it is strange that both of our stores (sandbox and production) complain when we try to do more than 5
